translator/source_code/deduplicate.py

```diff
@@ -0,0 +1,67 @@
+import json
```

Comment:
I don't see any unit tests for this file?

Reply:
Also, what are we de-duplicating? Are there duplicate messages we need to get rid of? I was under the impression that we would be merging our icone data into the cotrip data which would require finding the matching sets and combining data, not removing anything.

Reply:
That is exactly what was intended, de-duplicating is inaccurate. This is being put on hold because we do not have any duplicate iCone/cotrip messages, so this can be deleted